<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotifyKindsWidgetClass</class>
 <widget class="QWidget" name="NotifyKindsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>47</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grbKinds">
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chbRoster">
        <property name="text">
         <string>Roster</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbPopup">
        <property name="text">
         <string>Popup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbTray">
        <property name="text">
         <string>Tray</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbSound">
        <property name="text">
         <string>Sound</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbActivate">
        <property name="text">
         <string>Activate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
